srcdoc/configure/http3_directives.mt

```diff
@@ -124,6 +124,22 @@ See <a href="configure/http2_directives.html#http2-input-window-size"><code>http
 </p>
 ? })
 
+<?
+$ctx->{directive}->(
+    name    => "http3-max-concurrent-streaming-requests-per-connection",
+    levels  => [ qw(global) ],
+    default => 'http3-max-concurrent-streaming-requests-per-connection: 1',
+    desc    => <<'EOT',
+Maximum number of streaming requests to be handled concurrently within a single HTTP/3 connection.
+EOT
+)->(sub {
+?>
+<p>
+The value cannot exceed 256.
+</p>
+? })
```

Comment:
```suggestion
)->(sub {});
```

Let's remove this.

Maximum concurrency of 256 is an old limit in the HTTP/2 code, I doubt we have it any more, but moreover, there is no such restriction in our H3 code.